about/reports/2020_downloads_top_250_institutions.md

```diff
@@ -0,0 +1,264 @@
+| 216 | polimi.it | 0.08% | 25161                                        |
+| 217 | metu.edu.tr | 0.08% | 24846                                      |
+| 218 | jinr.ru | 0.08% | 24487                                          |
+| 219 | csic.es | 0.08% | 24373                                          |
+| 220 | nao.ac.jp | 0.08% | 24366                                        |
+| 221 | fu-berlin.de | 0.08% | 24155                                     |
+| 222 | uni-leipzig.de | 0.08% | 24118                                   |
+| 223 | ncku.edu.tw | 0.08% | 23941                                      |
+| 224 | utl.pt | 0.08% | 23938                                           |
+| 225 | upv.es | 0.08% | 23660                                           |
+| 226 | syracuse.edu | 0.07% | 23488                                     |
+| 227 | usst.edu.cn | 0.07% | 23352                                      |
+| 228 | iastate.edu | 0.07% | 23000                                      |
+| 229 | cnrs.fr | 0.07% | 22992                                          |
+| 230 | bath.ac.uk | 0.07% | 22963                                       |
+| 231 | ifpan.edu.pl | 0.07% | 22647                                     |
+| 232 | jyu.fi | 0.07% | 22283                                           |
+| 233 | ipm.ac.ir | 0.07% | 22117                                        |
+| 234 | keio.ac.jp | 0.07% | 22049                                       |
+| 235 | lut.ac.uk | 0.07% | 21859                                        |
+| 236 | ucm.es | 0.07% | 21841                                           |
+| 237 | adelaide.edu.au | 0.07% | 21605                                  |
+| 238 | umontreal.ca | 0.07% | 21523                                     |
+| 239 | tu-ilmenau.de | 0.07% | 21465                                    |
+| 240 | unipi.it | 0.07% | 21456                                         |
+| 241 | unifi.it | 0.07% | 21309                                         |
+| 242 | okstate.edu | 0.07% | 21139                                      |
+| 243 | uni-bremen.de | 0.07% | 20927                                    |
+| 244 | au.dk | 0.07% | 20906                                            |
+| 245 | oregonstate.edu | 0.07% | 20643                                  |
+| 246 | tu-chemnitz.de | 0.07% | 20641                                   |
+| 247 | leeds.ac.uk | 0.07% | 20588                                      |
+| 248 | pd.infn.it | 0.06% | 20417                                       |
+| 249 | roma1.infn.it | 0.06% | 20320                                    |
+| 250 | univ-lyon1.fr | 0.06% | 20101                                    |
+
+(\*) Some institutions' downloads come from multiple domains which have been combined to report an aggregate number:
+
+(\**) Some institutions' downloads were created via an alternative method due to anomalies and are noted by the parentheses around the rank number.
```

Comment:
If there are no ** data this line could be removed
```suggestion

```

Reply:
@bdc34 or @bmaltzan can you confirm which, if any, domains' downloads were found using alternate methods? If none, let's delete this line.

Reply:
In the past this is the list of institutions that required additional analysis. An example of this would be an institution that goes from the top 10 to position 600. In the past I've taken that list and investigated why and performed alternative analysis methods. An example of an alternative method would be to contact the institution for an IP range and do additional queries for those ranges. Another example would be to filter out automated traffic for an institution.

There aren't really any this year that have been like that.